### PR TITLE
update project schema to Xcode13 format. iOS Deployment target: iOS 14.0

### PR DESCRIPTION
**Pull request by theInternauts**
- Update project files to Xcode 13+
- Minimum Deployment Target iOS 14.0

### CODE REVIEW

**Comment by theInternauts** [comment]
If someone can confirm a local build on Xcode13+  This PR can be merged.

In the near future I want to setup [xcodegen](https://github.com/yonaskolb/XcodeGen) to help standardize our builds as we work together on this project

File: Podfile.lock [hunk 1]
```diff
@@ -115,4 +115,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: dcce34aea2296a4c3e247b2b2ff8a0e9a1cce2b9
 
-COCOAPODS: 1.11.2
+COCOAPODS: 1.10.1
```
Comment (zspencer) [inline Podfile.lock:118]: Is there a reason Cocoapods got downgraded?

**Review by zspencer — APPROVED** [review]
Love it! I did a quick pull and build and it looks great! I also [made a PR with the cocoapods update](https://github.com/zinc-collective/mp-shake-it-photo/pull/42); but I think it's 💀 so maybe don't merge my shit unless you think it's a good call?

File: Podfile.lock (hunk 1 above)
Comment (theInternauts) [reply Podfile.lock:118]: no reason.  I haven't dared to change my config for fear of more complications.  I will upgrade in the next PR.

**Pull request merged**